src/components/Dashboard/TodoList.vue

```diff
@@ -70,6 +70,13 @@
               <div class="list-item__subtitle">
                 <div v-if="task.subtitle" class="todo-status">
                   <span>{{task.subtitle}}</span>
+                  <div  v-if="isTypeNameRequest(task)">
+                    <v-btn icon class="nr-info-btn" color="blue darken-2" :ripple="false"
+                    @click="isNRDetailsVisible=!isNRDetailsVisible">
+                        <v-icon>mdi-information-outline</v-icon>
```

Comment:
Does this icon need a colour (like I added in my PR)?

Reply:
I think it looks fine.

Reply:
OK, thanks for checking.
Please add color to v-icon as I did in a prev PR (which you haven't rebased to yet). See https://github.com/bcgov/business-filings-ui/pull/64/files#diff-ab0d23117b3622c2916b8428c4342b11R118

Reply:
I dont think this is required as the color is applied to whole button itself. I had checked the vuetify sample of how to do that in a tile button. However I have added that